test/integration/util/util.go

```diff
@@ -431,15 +431,15 @@ func RemoveTaintOffNode(cs clientset.Interface, nodeName string, taint v1.Taint)
 
 // WaitForNodeTaints waits for a node to have the target taints and returns
 // an error if it does not have taints within the given timeout.
-func WaitForNodeTaints(cs clientset.Interface, node *v1.Node, taints []v1.Taint) error {
-	return wait.Poll(100*time.Millisecond, 30*time.Second, NodeTainted(cs, node.Name, taints))
+func WaitForNodeTaints(ctx context.Context, cs clientset.Interface, node *v1.Node, taints []v1.Taint) error {
```

Comment:
/cc @soltysh

As you just touched test/*.go utils.

This is an example where `tCtx ktesting.TContext` would be useful (IMHO) because it's one parameter instead of two (`ctx` and `cs`).

Reply:
I considered use `TestContext` to reduce the function parameters initially, the intension is similar to `ktesting.TContext`, but `(ctx context.Context, cs clientset.Interface)` is used across utils, I would like to keep them consistent. So the new work can be based on this PR as incremental improvements if we come to an agreement to use `ktesting.TContext`. 

I think **consistency is great for maintenance**.

Reply:
Yes, this is definitely something for the future.

The problem with getting to that future is that it can either be a sweeping change (not great for reviews and merging) or it can be incremental (not consistent while this is going on). Neither is great :cry:

Reply:
I prefer the latter with a suitable size.

Reply:
Do you want anything else to include in this PR? @pohly

Reply:
No, looks good.

/approve